ui/page/transaction/transactions_page.go

```diff
@@ -222,95 +222,138 @@ func (pg *TransactionsPage) loadTransactions(offset, pageSize int32) (interface{
-												return layout.E.Layout(gtx, func(gtx C) D {
-													// Show bottom divider for all rows except last
-													return layout.Inset{Left: values.MarginPadding56}.Layout(gtx, separator.Layout)
-												})
-											}),
-										)
+										wallTxs := pg.scroll.FetchedData().([]sharedW.Transaction)
+										return pg.transactionList.Layout(gtx, len(wallTxs), func(gtx C, index int) D {
+											row := components.TransactionRow{
+												Transaction: wallTxs[index],
+												Index:       index,
+											}
+											return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+												layout.Rigid(func(gtx C) D {
+													return components.LayoutTransactionRow(gtx, pg.Load, row, true)
+												}),
+												layout.Rigid(func(gtx C) D {
+													// No divider for last row
+													if row.Index == len(wallTxs)-1 {
+														return D{}
+													}
+
+													gtx.Constraints.Min.X = gtx.Constraints.Max.X
+													separator := pg.Theme.Separator()
+													return layout.E.Layout(gtx, func(gtx C) D {
+														// Show bottom divider for all rows except last
+														return layout.Inset{Left: values.MarginPadding56}.Layout(gtx, separator.Layout)
+													})
+												}),
+											)
+										})
 									})
 								})
 							})
 						})
 					})
 				}),
 				layout.Expanded(func(gtx C) D {
+					if pg.isHomepageLayout && len(pg.assetWallets) > 1 {
+						return pg.walletDropDown.Layout(gtx, 0, false)
+					}
+					return D{}
+				}),
+				layout.Expanded(func(gtx C) D {
+					if pg.showDisabledLayout {
+						return D{}
+					}
 					return pg.txTypeDropDown.Layout(gtx, 0, true)
 				}),
 			)
 		})
 	})
 
-	items := []layout.FlexChild{layout.Rigid(pg.pageTitle)}
-	if pg.WL.SelectedWallet.Wallet.GetAssetType() == utils.DCRWalletAsset {
-		// Layouts only supportted by DCR
-		line := pg.Theme.Separator()
-		line.Color = pg.Theme.Color.Gray3
-		items = append(items, layout.Rigid(line.Layout))
+	items := []layout.FlexChild{}
+	if pg.hasDCRwallet && pg.selectedWallet.GetAssetType() == utils.DCRWalletAsset {
+		// Layouts only supported by DCR
 		items = append(items, layout.Rigid(pg.sectionNavTab))
 	}
+
 	items = append(items, txlisingView)
+	if pg.isHomepageLayout {
+		return components.UniformPadding(gtx, func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx, items...)
+		})
+	}
 	return layout.Flex{Axis: layout.Vertical}.Layout(gtx, items...)
 }
 
-func (pg *TransactionsPage) layoutMobile(gtx layout.Context) layout.Dimensions {
+func (pg *TransactionsPage) sectionNavTab(gtx C) D {
+	return cryptomaterial.LinearLayout{
+		Width:       cryptomaterial.MatchParent,
+		Height:      cryptomaterial.WrapContent,
+		Orientation: layout.Horizontal,
+		Direction:   layout.Center,
+	}.Layout2(gtx, func(gtx C) D {
+		return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, pg.tab.Layout)
+	})
```

Comment:
I'll like to see a `cryptomaterial.CentralizeWidget(widget D)` method that uses a LinearLayout to do the centralization. We shouldn't be writing plenty code each time just to centralize one widget.

Reply:
I mean something like this:
```
func CentralizeWidget(gtx C, widget layout.Widget) D {
	return cryptomaterial.LinearLayout{
		Width:       cryptomaterial.MatchParent,
		Height:      cryptomaterial.WrapContent,
		Orientation: layout.Horizontal,
		Direction:   layout.Center,
	}.Layout2(gtx, widget)
}
```

Then here you use as:
```
cryptomaterial.CentralizeWidget(gtx, func(gtx C) D {
	return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, pg.tab.Layout)
})
```